source/jormungandr/tests/coverage_tests.py

```diff
@@ -46,6 +46,9 @@ def test_null_status_is_hidden(self):
         assert 'regions' in response
         assert len(response['regions']) == 1
         assert response['regions'][0]['id'] == 'main_routing_test'
+        assert 'last_load_at' in response['regions'][0]
```

Comment:
if the sort of the region has been asked, can you test it ? you might want to use a fixture with several region loaded (this one will hide the empty region :smile: )